src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -121,7 +140,7 @@ public int Capacity
         /// <returns>The index of the first occurrence of an item.</returns>
         public int IndexOf(T item)
         {
-            return Array.IndexOf(_items, item, 0, _size);
+            return Array.IndexOf(_items, item, 0, Count);
```

Comment:
I removed `_size` field. Now only property `public int Count { get; private set; }`. Somebody ask to make so. Return `_size` field?